DFC.ServiceTaxonomy.ApiFunction.Tests/ExecuteHttpTriggerTests.cs

```diff
@@ -0,0 +1,251 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using System.Web.Http;
+using Castle.Components.DictionaryAdapter;
+using DFC.ServiceTaxonomy.ApiFunction.Function;
+using DFC.ServiceTaxonomy.ApiFunction.Helpers;
+using DFC.ServiceTaxonomy.ApiFunction.Models;
+using FakeItEasy;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Internal;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Options;
+using Neo4j.Driver.V1;
+using Newtonsoft.Json;
+using Xunit;
+using ILogger = Microsoft.Extensions.Logging.ILogger;
+
+namespace DFC.ServiceTaxonomy.ApiFunction.Tests
```

Comment:
Nice and comprehensive testing of unhappy paths, but is missing happy path test(s).

Reply:
We now have a happy path test, that checks that a 200 response code is returned.